src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagService.kt

```diff
@@ -172,10 +171,30 @@ class ProductaanvraagService @Inject constructor(
                 titel = bijlage.titel
                 beschrijving = bijlage.beschrijving
             }.run {
-                zrcClientService.createZaakInformatieobject(this, ZAAK_INFORMATIEOBJECT_REDEN)
+                createZaakInformatieobject(this)
+            }
+        }
+
+    private fun pairBijlagenWithZaakIgnoringExceptions(bijlageURIs: List<URI>, zaakUrl: URI) =
+        bijlageURIs.map(drcClientService::readEnkelvoudigInformatieobject).forEach { bijlage ->
+            ZaakInformatieobject().apply {
+                informatieobject = bijlage.url
+                zaak = zaakUrl
+                titel = bijlage.titel
+                beschrijving = bijlage.beschrijving
+            }.runCatching {
+                createZaakInformatieobject(this)
+            }.onFailure {
+                LOG.log(Level.WARNING, "Failed to pair bijlagen '${bijlage.url}' with zaak url '$zaakUrl'", it)
             }
         }
 
+    private fun createZaakInformatieobject(zaakInformatieobject: ZaakInformatieobject) {
```

Comment:
Not sure this function has any value? It's just one call with some debug lines. I would not make this into a function at all. Just makes code harder to read I feel.